### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by adrianne-helpscout**
Adding a note to the changelog about our move from support via GitHub Issues to our customer support queue.

### CODE REVIEW

**Review by keishaj-creates — COMMENTED** [review]
Do we need to change the date from Feb 4 to today?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 To install Beacon, see the [documentation](https://developer.helpscout.com/beacon-2/mobile/).
 
+## Change to developer support (Febuary 4, 2022)
+
+
```
Comment (ScottPetit) [inline CHANGELOG.md:8]:
```suggestion
Not a new release, per se, but we have migrated support from GitHub Issues in this repository to our customer support team. Links in earlier changelog entries that pointed to an issue will no longer work, you will get a 404 when following them. For additional information on a previous release, github issue, or any other request for support please reach out to us via [help@helpscout.com](mailto:help@helpscout.com?subject=%5BiOS%20Beacon%20SDK%5D%20Issue%20report&body=Technical%20details%0A%0A-%20Beacon%20version%3A%0A-%20iOS%20OS%20version%3A%0A-%20Device%20make%2Fmodel%3A%0A-%20Impacted%20user%20count%3A%0A%0AActual%20outcome%0A%0A%20%20%20%20Beacon%20did%20or%20did%20not...%0A%0AExpected%20outcome%0A%0A%20%20%20%20Beacon%20should...%0A%0ASteps%20to%20reproduce%0A%0A%20%20%20%201.%20%E2%80%A6%0A%20%20%20%202.%20%E2%80%A6%0A%20%20%20%203.%20%E2%80%A6%0A%0AScreenshots%2FScreen%20Recording%0A%0A%20%20%20%20Please%20add%20any%20images%20or%20videos%20to%20support%20your%20report.%0A%0AAdditional%20Information%0A%0A%20%20%20%20Please%20attach%20any%20crash%20logs)

```

**Review by ScottPetit — CHANGES_REQUESTED** [review]
It seems like the intent of this was to align the "Not a new release" paragraph below the `Change to developer support` section.  It seems to have snuck in under the 2.2.4 release section though.

**Comment by adrianne-helpscout** [comment]
@keishaj-creates We made the change on Feb 4th; so, I think best to leave that as the date as this should be a log of changes that were made along with their effective dates, and not reflect the date when we made the changelog change.

**Review by ScottPetit — APPROVED** [review]
Looks good to me.

**Review by keishaj-creates — APPROVED** [review]

**Pull request merged**